.github/workflows/release-wc-beta-tester.yml

```diff
@@ -0,0 +1,33 @@
+name: WooCommerce Beta Tester Release
+
+on:
+    workflow_dispatch:
+        inputs:
+            version:
+                description: 'The version number for the release'
+                required: true
+
+jobs:
+    release:
+        name: Run release scripts
+        runs-on: ubuntu-20.04
+        permissions:
+            contents: write
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Setup WooCommerce Monorepo
+              uses: ./.github/actions/setup-woocommerce-monorepo
+
+            - name: Build WooCommerce Beta Tester Zip
+              working-directory: plugins/woocommerce-beta-tester
+              run: pnpm build:zip
+
+            - name: Create release
+              id: create_release
+              uses: woocommerce/action-gh-release@master
```

Comment:
I created a fork of the action, originally in my own account to test and add changes to support changing allowing marking a release as "latest" or not. With this fork it now defaults to not marking the release as latest which you can see in my test run in the PR description.